public/app/features/alerting/unified/utils/amroutes.ts

```diff
@@ -237,8 +237,8 @@ export const objectMatchersToString = (matchers: ObjectMatcher[]): string[] => {
   });
 };
 
-export const repeatIntervalValidator = (repeatInterval: string, groupInterval: string) => {
-  if (repeatInterval.length === 0) {
+export const repeatIntervalValidator = (repeatInterval: string, groupInterval?: string) => {
```

Comment:
This probably needs to change; we shouldn't pass validation for the repeat interval if no group interval was specified since that's an optional field.

I suggest defaulting the `groupInterval = ''` and removing the additional check for `groupInterval`

Reply:
Resolved in https://github.com/grafana/grafana/pull/79493/commits/f24bf2fd4ffb221ae1c46c1c3d604c153cebe86b